src/commands/init.ts

```diff
@@ -79,7 +79,7 @@ export default class Init extends Command {
       cli.action.stop();
 
       cli.action.start('Creating files');
-      copyFiles(files, options, this.error);
+      copyFiles(files, options).catch(e => this.error(e, { exit: 2 }));
```

Comment:
Shouldn't there be an `await` here now that the function is `async`?

Reply:
Good catch!